app/views/root/_error_page.html.erb

```diff
@@ -26,6 +26,12 @@
       </div>
     </article>
   </div>
+
+<% if status_code %>
+  <script type="text/javascript">
+    GOVUK.httpStatusCode = '<%= status_code %>';
```

Comment:
This code as is will cause `Uncaught ReferenceError: GOVUK is not defined`. If we want to attach the status code onto the `GOVUK` object you will need the line 

```javascript
window.GOVUK = window.GOVUK || {};
```

before this. Alternatively for simplicity I can't see any problem with setting this onto the window object.